internal/testutil/acc/cluster.go

```diff
@@ -47,11 +49,15 @@ func GetClusterInfo(tb testing.TB, req *ClusterRequest) ClusterInfo {
 		}
 	}
 	projectID = ProjectIDExecution(tb)
-	clusterTerraformStr, clusterName, err := ClusterResourceHcl(projectID, req)
+	return GetClusterInfoWithProject(tb, req, projectID, DefaultClusterResourceSuffix)
+}
+
+func GetClusterInfoWithProject(tb testing.TB, req *ClusterRequest, projectID, resourceSuffix string) ClusterInfo {
```

Comment:
what about adding a new "projectID" or similar attribute to ClusterRequest and have only GetClusterInfo instead of GetClusterInfoWithProject?

Reply:
adding a new "projectID" to existing function might force having to pass nil in the majority of the usages. I like the idea of a seperate function but name was slightly confusing at first, maybe `GetClusterInfoUsingExistingProject` might be more clear?

Reply:
i mean to have it in the cluster request struct so don't need to set it if don't need it. but i'm ok to leave both functions

Reply:
@lantoli @AgustinBettati 
Good discussion! Updated in [2e36af0](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2412/commits/2e36af0f89f87d6adadd8d022dcb29cb36dd1b59) WDYT?